### PR TITLE
[1892] Update privacy policy content

### PR DESCRIPTION
**Pull request by darokel**
### Context

- https://trello.com/c/D7EAYrTA/1892-update-the-content-for-the-privacy-policy

### Changes proposed in this pull request

- Extract content into yaml to be consistent with other static pages and make it easier to switch to alternate lang in the future
- Update the content

### Guidance to review

https://rtt-review-pr-942.herokuapp.com/privacy-policy

### CODE REVIEW

File: config/locales/pages/privacy_policy/en.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  pages:
+    privacy_policy:
+      body_html: "
+      <p class='govuk-body'>This service allows you to share and access trainee data with the Department for Education (DfE).</p>
+
+      <p class='govuk-body'>By using the service, you’ll be sharing your data and the data of trainees.</p>
+
+      <div class='govuk-inset-text'>This privacy policy is about your personal data and Initial Teacher Training (ITT) trainee data.</div>
+
+      <h2 class='govuk-heading-m '>In this privacy policy</h2>
```
Comment (edwardhorsford) [inline config/locales/pages/privacy_policy/en.yml:11]:
```suggestion
      <h2 class='govuk-heading-m'>In this privacy policy</h2>
```

File: config/locales/pages/privacy_policy/en.yml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        <li>create log files of your activity and any error messages you come across</li>
+      </ul>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'>If you take part in user research, you may choose to share additional information about you.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'>We use Google Analytics software to collect information about how you use Register trainee teachers. This includes IP addresses. It helps us to improve the service.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'>We do not store your personal information through Google Analytics. The data is anonymised before being used for analytics processing.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'><a href='/cookies' class='govuk-link'>Find out more about our cookie policy</a></p>
```
Comment (edwardhorsford) [inline config/locales/pages/privacy_policy/en.yml:47]: Is it intentional that these all have a margin override?

File: config/locales/pages/privacy_policy/en.yml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+      <h2 class='govuk-heading-m' id='our-lawful-basis-for-processing-your-personal-data'>Our lawful basis for processing your personal data</h2>
+
+      <p class='govuk-body'>Running and improving the service is a public task, as defined in data protection law. However, we ask for your consent to take part in user research.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'><a href='https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/' class='govuk-link'>Read about consent and public task on the Information Commissioner's Office (ICO) website</a></p>
```
Comment (edwardhorsford) [inline config/locales/pages/privacy_policy/en.yml:53]: Is this margin override needed? Same q for all overrides.

**Review by gpeng — APPROVED** [review]

File: config/locales/pages/privacy_policy/en.yml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+      <p class='govuk-body'>You can also <a href='https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen' class='govuk-link'>get in touch with the Department for Education’s data protection officer through the online contact form</a>.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'><a href='https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter' class='govuk-link'>Refer to the Department for Education’s personal information charter to read more about the standards you can expect from us when we look after your personal data</a></p>
+
+      <h2 class='govuk-heading-m' id='keep-up-to-date'>Keep up to date</h2>
+
+      <p class='govuk-body'>We periodically review this policy. We’ll notify you if we change how we use your personal data.</p>
+
+      <p class='govuk-body govuk-!-margin-bottom-7'>It was last updated on 1 June 2021.</p>
```
Comment (edwardhorsford) [inline config/locales/pages/privacy_policy/en.yml:144]: Update?

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

File: config/locales/pages/privacy_policy/en.yml (hunk 3 above)
Comment (darokel) [reply config/locales/pages/privacy_policy/en.yml:53]: This html was taken from the prototype

File: config/locales/pages/privacy_policy/en.yml (hunk 4 above)
Comment (darokel) [reply config/locales/pages/privacy_policy/en.yml]: done https://github.com/DFE-Digital/register-trainee-teachers/pull/942/commits/742b292841880f2239a6bd0eb7f99c90054e0680

File: config/locales/pages/privacy_policy/en.yml (hunk 3 above)
Comment (darokel) [reply config/locales/pages/privacy_policy/en.yml]: removed https://github.com/DFE-Digital/register-trainee-teachers/pull/942/commits/742b292841880f2239a6bd0eb7f99c90054e0680

File: config/locales/pages/privacy_policy/en.yml (hunk 2 above)
Comment (darokel) [reply config/locales/pages/privacy_policy/en.yml]: removed https://github.com/DFE-Digital/register-trainee-teachers/pull/942/commits/742b292841880f2239a6bd0eb7f99c90054e0680

File: config/locales/pages/privacy_policy/en.yml (hunk 1 above)
Comment (darokel) [reply config/locales/pages/privacy_policy/en.yml]: done https://github.com/DFE-Digital/register-trainee-teachers/pull/942/commits/742b292841880f2239a6bd0eb7f99c90054e0680

**Review by edwardhorsford — APPROVED** [review]

**Comment by darokel** [comment]
[Update the content for the privacy policy](https://trello.com/c/D7EAYrTA/1892-update-the-content-for-the-privacy-policy)

**Pull request merged**